pkg/storage/mvcc_test.go

```diff
@@ -2797,7 +2796,9 @@ func TestMVCCReverseScanSeeksOverRepeatedKeys(t *testing.T) {
 // The bug happened in this scenario.
 // (1) reverse scan is positioned at the range's smallest key and calls `prevKey()`
 // (2) `prevKey()` peeks and sees newer versions of the same logical key
-//	   `iters_before_seek_-1` times, moving the iterator backwards each time
+//
```

Comment:
Was this indented due to a new gofmt? If so, should we revert it?

Reply:
This was unintentional. Reverted this part.